src/main/java/io/ddavison/conductor/Locomotive.java

```diff
@@ -66,6 +79,34 @@ public void init() {
         }
     }
 
+
+    protected DesiredCapabilities onCapabilitiesCreated(DesiredCapabilities desiredCapabilities) {
+        return desiredCapabilities;
+    }
+
+    private DesiredCapabilities getCapabilities(ConductorConfig configuration) {
```

Comment:
This private method does not seem to add any functionality. Could the calls to this be replaced by `buildCapabilities` and this method removed?

Reply:
Oh yes, removed 👍